packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-usage-tile/DatacentreUsageTile.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import {
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import DatacentreUsageTile from './DatacentreUsageTile.component';
+import { datacentreList } from '../../../../mocks/vcd-organization/vcd-datacentre.mock';
+
+describe('DatacentreUsageTile component unit test suite', () => {
+  it('should define all sections with correct typo', () => {
+    // when
+    const { getByText } = render(
+      <DatacentreUsageTile vcdDatacentre={datacentreList[0]} />,
+    );
+
```

Comment:
```suggestion
    const { asFragment } = render(
      <DatacentreUsageTile vcdDatacentre={datacentreList[0]} />,
    );
    expect(asFragment()).toMatchSnapshot();
```

You could do this also if you want to check the component implementation